### ISSUE TITLE
Cannot open Opera browser

### ISSUE THREAD

**Issue by dongdn** [SeleniumHQ/selenium#5266]
## Meta -
OS:  Windows 10 64bit

Selenium Version:  3.8.0

Browser:  Opera

Browser Version:  49.0.2725.64 (PGO) 

## Expected Behavior -
Open Opera browser
## Actual Behavior -
Throw an exception as below:

```
Traceback (most recent call last):
  File "c:/Users/dongdn/Desktop/Robot/RobotLibrary/test.py", line 3, in <module>
    browser = webdriver.Opera();
  File "C:\Python36\lib\site-packages\selenium\webdriver\opera\webdriver.py", line 78, in __init__
    service_log_path=service_log_path)
  File "C:\Python36\lib\site-packages\selenium\webdriver\opera\webdriver.py", line 57, in __init__
    service_log_path=service_log_path)
  File "C:\Python36\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 75, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 154, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 243, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "C:\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 311, in execute
    self.error_handler.check_response(response)
  File "C:\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 237, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot parse capability: operaOptions
from unknown error: must be a dictionary
  (Driver info: OperaDriver=2.32 (cfa164127aab5f93e5e47d9dcf8407380eb42c50),platform=Windows NT 10.0.14393 x86_64)
```
## Steps to reproduce -
1. Run the following code.

```
from selenium import webdriver
browser = webdriver.Opera()
browser.get('https://google.com')
```

2. The exception will be thrown.

**Comment by xvfionn** [comment]
This is a regression; it works in 3.6.0, but not subsequent releases.

**Comment by lmtierney** [comment]
This was fixed in 8f61013 and will be in the next release. Thanks!
